packages/frontend2/svgr.d.ts

```diff
@@ -5,7 +5,6 @@ declare module '*.svg' {
 }
 
 declare module '*.svg?url' {
-  // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  const content: any
+  const content: string
```

Comment:
I checked it recently and it's not a string. Svgr also recommends typing it as any.
```suggestion
  // eslint-disable-next-line @typescript-eslint/no-explicit-any
  const content: any
```